src/front/DataInspector.js

```diff
@@ -124,3 +141,24 @@ L.Kosmtik.DataInspector = L.Class.extend({
     }
 
 });
+
+L.FormBuilder.LabeledCheckBox = L.FormBuilder.CheckBox.extend({
```

Comment:
@yohanboniface if you do add it to L.FormBuilder: within the div, I nest the checkbox and span.layer-name inside another element, label.layer-label. This makes it possible to toggle the checkbox by clicking the text. If L.FormBuilder.CheckBox is used elsewhere, the new nesting of the elements might mess up some styles.